plugins/labhub.py

```diff
@@ -222,6 +222,18 @@ def create_issue_cmd(self, msg, match):
                 target=user,
             )
 
+    @staticmethod
+    def profile_link(msg):
+        gitter_username = msg.frm.nick
+        pattern = '^(.*)(_.*?)$'
+        if '_gitlab' in gitter_username:
+            username = re.match(
+                pattern, gitter_username.split('@')[-1]).group(1)
+            return 'https://gitlab.com/{user}'.format(user=username)
```

Comment:
Btw you sure it reads so well providing the full url? I would maybe do something like `@<username> (GitLab)` or similar

Reply:
Ah okay now I see... It's otherwise linking to the user. Is it possible to override the link via `[]()` syntax?

Reply:
Yeah, right now it would look something like:

Opened by @https://gitlab.com/abhishalya at Gitter.

A simple change to https://github.com/coala/corobo/blob/master/plugins/labhub.py#L208 would fix this.

Reply:
Yeah we could override it.We could even do something like 
```hyperlink_format = '<a href="{link}">{text}</a>```
```hyperlink_format.format(link='https://gitlab.com/sladyn98', text='@sladyn98')```

Reply:
No, we return a markdown string to gitter.